### PR TITLE
v8.0.0

### PR DESCRIPTION
**Pull request by deepa-iOS-SDK**

### CODE REVIEW

**Comment by RakshithaAcharya** [comment]
@deepa-iOS-SDK  lets create a copy of dev branch first as 7.x.x-above if we need to release on earlier series

**Comment by deepa-iOS-SDK** [comment]
> @deepa-iOS-SDK lets create a copy of dev branch first as 7.x.x-above if we need to release on earlier series

Done

File: Pod/Classes/SEGMoEngageInitializer.m [hunk 1]
```diff
@@ -10,49 +10,49 @@
 #import <MoEngageSDK/MoEngageSDK.h>
 
 
-static MOSDKConfig* currentSDKConfig = nil;
+static MoEngageSDKConfig* currentSDKConfig = nil;
 static NSString * const segmentVersion = @"7.1.0";
```
Comment (RakshithaAcharya) [inline Pod/Classes/SEGMoEngageInitializer.m:14]: @deepa-iOS-SDK  update the segment version

**Review by RakshithaAcharya — CHANGES_REQUESTED** [review]

File: Pod/Classes/SEGMoEngageInitializer.m (hunk 1 above)
Comment (deepa-iOS-SDK) [reply Pod/Classes/SEGMoEngageInitializer.m:14]: Done

**Review by RakshithaAcharya — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 Change Log
 ==========
+Version 8.0.0 *(4th January, 2023)*
```
Comment (RakshithaAcharya) [inline CHANGELOG.md:3]: @deepa-iOS-SDK update the date if required

**Review by RakshithaAcharya — APPROVED** [review]

File: Segment-MoEngage.podspec [hunk 3]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = "Segment-MoEngage"
-  s.version          = "7.5.0"
+  s.version          = "8.0.0"
```
Comment (umangmoe) [inline Segment-MoEngage.podspec]: This should be a separate commit in the master branch. should not be changed here.

**Review by umangmoe — APPROVED** [review]
